src/main/java/com/wolox/training/controllers/BookController.java

```diff
@@ -1,15 +1,61 @@
 package com.wolox.training.controllers;
 
-import org.springframework.stereotype.Controller;
-import org.springframework.ui.Model;
+import com.wolox.training.exceptions.books.BookIdMismatchException;
+import com.wolox.training.exceptions.books.BookNotFoundException;
+import com.wolox.training.models.Book;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.DeleteMapping;
 import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+import com.wolox.training.repositories.BookRepository;
 
-@Controller
+@RestController
+@RequestMapping("/api/books")
 public class BookController {
-  @GetMapping("/book")
-  public String greeting(@RequestParam(name="name", required=false, defaultValue="World") String name, Model model) {
-    model.addAttribute("name", name);
-    return "book";
+
+  @Autowired
+  private BookRepository bookRepository;
+
+  @GetMapping
+  public Iterable findAll() {
+    return bookRepository.findAll();
+  }
+
+  @GetMapping("/title/{bookTitle}")
```

Comment:
Usually it's more restful to use queryparams for this kind of searchs, something like 
`https://localhost:8080/api/books?title=mititulo`
Here you have a [link](https://www.baeldung.com/spring-request-param#a-simple-mapping) that shows how to use them